guides/protocol.html

```diff
@@ -34,7 +34,8 @@
     <li><a id="guides-claims"   href="/guides/claims.html">Name Claims</a></li>
     <li><a id="guides-auctions"   href="/guides/auctions.html">Name Auctions</a></li>
     <li><a id="guides-wallet"   href="/guides/wallet.html">Wallet Setup</a></li>
-    <li><a id="guides-protocol"  class="active"  href="/guides/protocol.html">Protocol Summary</a></li>
+    <li><a id="guides-resource-records"   href="/guides/resource-records.html">Handshake Resource Records</a></li>
+    <li><a id="guides-protocol"   href="/guides/protocol.html">Protocol Summary</a></li>
```

Comment:
this should have been fixed: 

https://github.com/handshake-org/handshake-org.github.io/commit/17d730414a587ec1bc8c9b2c990304b82659e6b8

you might need to rebase on master.

also, I think best practice for this repo should be to never include the build in the PR.
Let's run the build on a separate commit after the markdown file is merged to master.